src/main/java/model/data/DataTable.java

```diff
@@ -25,7 +27,7 @@ public DataTable() {
 	 */
 	public DataTable(String name, List<DataRow> rows, List<DataColumn> columns) {
 		this();
-		this.rows = new ArrayList<>(rows);
+		this.rows = rows;
```

Comment:
je hebt gelijk, had het tijdelijk aangepast in een poging om achter te halen waar iets fout ging, maar ben het zo te zien vergeten terug te veranderen.